cmd/soroban-cli/src/commands/mod.rs

```diff
@@ -45,11 +45,10 @@ Full CLI reference: https://github.com/stellar/soroban-tools/tree/main/docs/soro
 #[derive(Parser, Debug)]
 #[command(
     name = "soroban",
-    version = version::short(),
-    long_version = version::long(),
     about = ABOUT,
     long_about = ABOUT.to_string() + LONG_ABOUT,
     disable_help_subcommand = true,
+    disable_version_flag = true,
```

Comment:
We used to disable `--version` / `-v` and added it back in https://github.com/stellar/soroban-tools/pull/484

I agree that removing the difference between `version` and `long_version` seems useful to cut down on troubleshooting convo length, but personally find it unexpected when a CLI doesn't support `--version` or `-v`.

Reply:
> I agree that removing the difference between `version` and `long_version` seems useful to cut down on troubleshooting convo length

Good point, this is really the most important thing to change. If there are extra other ways to get at version it doesn't matter so much. I'll reduce this change to that then I think we should move on to other things.